xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2352,6 +2352,7 @@ xe.admin.groups.addUser=Users to add
 xe.admin.groups.addUser.submit=Add
 xe.admin.groups.addGroup=Subgroups to add
 xe.admin.groups.addSuccess=Members successfully added
+xe.admin.groups.addIgnored=Members successfully added but some groups have been ignored ({0})
```

Comment:
So here the naming convention of https://dev.xwiki.org/xwiki/bin/view/Community/L10N%20Conventions is not respected. I guess if we change it we should also migrate the other ones in the future. You can check https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HMovingTranslations for this.

Reply:
Updated to `web.groups.administration.groupsIgnored`
- Declaration: https://github.com/xwiki/xwiki-platform/pull/1374/files#diff-fffbf37334c4d7a178b13f4ccc313f86R2358
- Use: https://github.com/xwiki/xwiki-platform/pull/1374/files#diff-675b8cf1e4313bb08807f805a03daa40R50

Reply:
AFAIK we don't use the `web` prefix.

Reply:
Note: I see `web` was used at https://github.com/xwiki/xwiki-platform/commit/749f52d0870e7014142890b10045afcedb3dd252#diff-cf04ec11293bf524b894888450e2230cR357 (but that's a very old commit). So not sure but needs to be checked and decided.

Reply:
The most recent use of the web prefix are from 2020 (https://github.com/xwiki/xwiki-platform/commit/e0b57863b3d8ff57e1a07e060d763b88d1baad3a#diff-fffbf37334c4d7a178b13f4ccc313f86R1669) but I agree that they are rarely used recently. Most of them have been introduced in 2015.

Would `groups.administration.groupsIgnored` be acceptable even though we don't have a module named `xwiki-platform-groups` yet?

Reply:
This commit seems to be a move of existing code so that translation key is much older.